workflow/defaults/settings.yaml

```diff
@@ -44,7 +44,13 @@ default_settings: &default_settings
 
   cplflx: .false.
   cplice: !calc cplflx
-  cpl: .false.
+  cpl: !FirstTrue
+    - when: !calc doc.settings.cplflx==".true."
+      do: .true.
+    - when: !calc doc.settings.cplice==".true."
```

Comment:
- when: !calc doc.settings.cplwav==".true."

cpl should also be true if cplwav is true and cplflx is false.  Same for cplchem

Reply:
@JessicaMeixner-NOAA The updating of cplwav is part of a systematic change that I've put together in PR#81. In other words, your suggestion is to essentially merge this one with PR#81 ( which you suggest we should discard?). This PR is branched from the trunk (feature/coupled-crow) which doesn't even have wave components in the configuration system yet. Please let me know what I should do.